src/response.rs

```diff
@@ -481,6 +481,17 @@ impl Response {
         }
     }
 
+    pub fn into_vec(self) -> Result<Vec<u8>, Error> {
+        let size = self
+            .header("Content-Type")
```

Comment:
Is this supposed to be "Content-Length"?
Also, what happens if e.g. "Transfer-Encoding: chunked" is used?
Using into_reader seems more convenient.

Reply:
> Is this supposed to be "Content-Length"?

Oops it is :)

> Also, what happens if e.g. "Transfer-Encoding: chunked" is used?

Well in that case more allocations will be made, this is just a small optimization